arch/arm/soc/nordic_nrf/nrf52/Kconfig.board

```diff
@@ -0,0 +1,21 @@
+# Kconfig.board - Nordic Semiconductor nRF52 MCU line
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+config GPIO_AS_PINRESET
```

Comment:
We should put these in  arch/arm/soc/nordic_nrf/nrf52/Kconfig.soc as the code that implements them exists in arch/arm/soc/nordic_nrf/nrf52/soc.c